.travis.yml

```diff
@@ -1,25 +1,18 @@
 language: scala
-matrix:
-  include:
-    - scala: 2.11.12
-      jdk: oraclejdk8
-    - scala: 2.12.6
-      jdk: oraclejdk8
+
+jdk: oraclejdk8
+
+env:
+  - SBT_CMD="scalafmtSbtCheck scalafmtCheck test:scalafmtCheck scalastyle test:scalastyle"
+  - SBT_CMD="coverage +test coverageReport coverageAggregate"
```

Comment:
The 2nd line overwrites the first?

Reply:
this will build a env matrix; those `SBT_CMD` will run in parallel.